### PR TITLE
Diffprivlib version 0.2.0

### PR DESCRIPTION
**Pull request by naoise-h**

### CODE REVIEW

File: diffprivlib/mechanisms/exponential.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def _get_utility(self, value1, value2):
             return 0
 
         if value1 > value2:
-            return self._get_utility(value2, value1)
+            return self._get_utility(value1=value2, value2=value1)
 
         return self._utility_values[value1 + "::" + value2]
```
Comment (stefano81) [inline diffprivlib/mechanisms/exponential.py]: This could be a tuple

File: tests/tools/test_mean.py [hunk 2]
```diff
@@ -52,3 +52,65 @@ def test_large_epsilon_axis(self):
...
+
+        for i in range(res.shape[0]):
+            self.assertAlmostEqual(res[i], res_dp[i], delta=0.01)
+
+    def test_nan(self):
+        a = np.random.random((5, 5))
+        a[2, 2] = np.nan
+
+        res = nanmean(a, range=1)
+        self.assertFalse(np.isnan(res))
```
Comment (stefano81) [inline tests/tools/test_mean.py]: I would suggest a different file

File: tests/tools/test_std.py [hunk 3]
```diff
@@ -53,3 +53,67 @@ def test_large_epsilon_axis(self):
...
+        for i in range(res.shape[0]):
+            self.assertAlmostEqual(res[i], res_dp[i], delta=0.01)
+
+    def test_nan(self):
+        a = np.random.random((5, 5))
+        a[2, 2] = np.nan
+
+        res = nanstd(a, range=1)
+        self.assertFalse(np.isnan(res))
+
```
Comment (stefano81) [inline tests/tools/test_std.py]: As per mean, IMHO better in a distinct file

File: tests/tools/test_var.py [hunk 4]
```diff
@@ -51,3 +51,64 @@ def test_large_epsilon_axis(self):
...
+
+        for i in range(res.shape[0]):
+            self.assertAlmostEqual(res[i], res_dp[i], delta=0.01)
+
+    def test_nan(self):
+        a = np.random.random((5, 5))
+        a[2, 2] = np.nan
+
+        res = nanvar(a, range=1)
+        self.assertFalse(np.isnan(res))
```
Comment (stefano81) [inline tests/tools/test_var.py]: I start seeing a pattern here

File: setup.py [hunk 5]
```diff
@@ -1,7 +1,24 @@
+import codecs
+import os
+
 import setuptools
 
-with open("README.md", "r") as fh:
-    long_description = fh.read()
+
+def read(rel_path):
```
Comment (stefano81) [inline setup.py]: rel_path -> file path

File: diffprivlib/mechanisms/exponential.py (hunk 1 above)
Comment (stefano81) [inline diffprivlib/mechanisms/exponential.py]: Suggestion to replace string keys with tuples

Comment (naoise-h) [reply diffprivlib/mechanisms/exponential.py]: Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/pull/13/commits/6b3183bd3dd807c4e2c3416358c0b95c875692fd

File: tests/tools/test_mean.py (hunk 2 above)
Comment (naoise-h) [reply tests/tools/test_mean.py]: Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/pull/13/commits/81161c91ae332cded7c897818f9794a1d49373da

File: tests/tools/test_std.py (hunk 3 above)
Comment (naoise-h) [reply tests/tools/test_std.py]: Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/pull/13/commits/81161c91ae332cded7c897818f9794a1d49373da

File: tests/tools/test_var.py (hunk 4 above)
Comment (naoise-h) [reply tests/tools/test_var.py]: Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/pull/13/commits/81161c91ae332cded7c897818f9794a1d49373da

File: setup.py (hunk 5 above)
Comment (naoise-h) [reply setup.py]: Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/pull/13/commits/ddb54916c91afa3a4513320edfb57daf1ebcf8e5

File: diffprivlib/mechanisms/exponential.py (hunk 1 above)
Comment (naoise-h) [reply diffprivlib/mechanisms/exponential.py]: Thanks for the suggestion, it has been fixed in https://github.com/IBM/differential-privacy-library/commit/6b3183bd3dd807c4e2c3416358c0b95c875692fd

**Pull request merged**